src/GuestConfiguration/GuestConfiguration.Test/ScenarioTests/GetVmGuestPolicyStatusHistoryTests.ps1

```diff
@@ -32,8 +32,8 @@ Get guest configuration policy status history by Vm name scope
 #>
 function Get-AzVMGuestPolicyStatusHistory-VmNameScope_Custom
 {
-	$rgName = "amits-test"
-	$vmName = "amits-winser6"
+	$rgName = "aashishCustomrole7ux"
```

Comment:
You should not hard code resource group and vm names - you should use the existing infrastructure to generate a random name for this test

Reply:
I will create a new PR to change the way tests are written for GCRP cmd-lets.
Wont fix in this PR.